plsync/sites.py

```diff
@@ -83,6 +83,7 @@
     makesite('atl05','67.106.215.192', '2610:0018:0111:C002::','Atlanta_GA', 'US', 33.636700, -84.428100, user_list, count=4, arch='x86_64', nodegroup='MeasurementLabCentos'),
     makesite('ath01','83.212.4.0',     '2001:648:2ffc:2101::', 'Athens', 'GR', 37.936400, 23.944400, user_list, nodegroup='MeasurementLabCentos'),
     makesite('ath02','83.212.5.128',   '2001:648:2ffc:2102::', 'Athens', 'GR', 37.936400, 23.944400, user_list, nodegroup='MeasurementLabCentos'),
+    makesite('ath03','193.201.166.128',   '2001:648:25e0::', 'Athens', 'GR', 37.936400, 23.944400, user_list, count=4, v6gw=2001:648:25e0::129 nodegroup='MeasurementLabCentos'),
```

Comment:
I noticed you've got count=4 specified. Can I assume that this is because we are going to combine the hardware from the old site into a new 4-node site?

Small nit: could you fix the spacing so that it lines up with the other rows through the city name, just for consistency?